src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -345,7 +345,7 @@ def send_command(self, command: str) -> CommandResult:
             ssm_waiter.wait(
                 InstanceId=self.instance_id,
                 CommandId=command_id,
-                WaiterConfig={"Delay": 5, "MaxAttempts": 30},
+                WaiterConfig={"Delay": 5, "MaxAttempts": 36},
```

Comment:
This would increase the timeout for SSM commands run by all tests. Increasing this could push our tests to run longer - especially in the case where the worker is unresponsive and increasing the time until these commands can fail.

I wonder if it would be better to make the waiter config a parameter to the `send_command` method. Then each caller can adjust the timeouts based on the command being run.

What do you think?